backend/prompts/prompting.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import os
+
+from jinja2 import Environment, FileSystemLoader
+
+
+class PromptEngine:
+    """
+    Class to handle loading and populating Jinja2 templates for prompts.
+    """
```

Comment:
We haven't used docstrings anywhere else. Should be consistent across the board. Either by adding them in or removing them here

Reply:
I agree, removed also